pkg/api/vminfo.go

```diff
@@ -328,14 +328,27 @@ type ScalingConfig struct {
 	// this field is left out the settings will fall back on the global default.
 	LoadAverageFractionTarget *float64 `json:"loadAverageFractionTarget,omitempty"`
 
-	// MemoryUsageFractionTarget sets the desired fraction of current memory that
-	// we would like to be using. For example, with a value of 0.7, on a 4GB VM
-	// we'd like to be using 2.8GB of memory.
+	// MemoryUsageFractionTarget sets the maximum fraction of current memory that we'd like to allow
+	// to be allocated (exactly MemTotal - MemAvailable, or roughly the memory that postgres has
+	// allocated [notably excluding LFC]).
```

Comment:
```suggestion
	// MemoryUsageFractionTarget sets the maximum fraction of total memory that postgres allocations (MemoryUsage) must fit into. 
	// This doesn't count the LFC memory.
```

I think the current explanation is a bit overloaded.